tests/func/experiments/test_checkpoints.py

```diff
@@ -143,7 +143,7 @@ def test_resume_branch(tmp_dir, scm, dvc, checkpoint_stage, workspace):
         checkpoint_stage.addressing,
         checkpoint_resume=branch_rev,
         params=["foo=100"],
-        tmp_dir=not workspace,
+        tmp_dir=True,  # not workspace,
```

Comment:
Looks like preceeding `apply` doesn't perform `dvc checkout` correctly, because checkout things that existing `foo` is already `2` instead of `4` because state db tells it so. Investigating if this is my patch or a problem with state 😱